opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -76,11 +76,15 @@
 class Link:
     """A link to a `Span`."""
 
+    empty_attributes = {}  # type: typing.Dict[str, types.AttributeValue]
```

Comment:
This can't be mutable since we expose it below, changing one (initially empty-attributed) link's attributes would affect others'.